src/main/java/com/cleanroommc/groovyscript/compat/mods/artisanworktables/Tables.java

```diff
@@ -0,0 +1,458 @@
+package com.cleanroommc.groovyscript.compat.mods.artisanworktables;
+
+import com.cleanroommc.groovyscript.GroovyScript;
+import com.cleanroommc.groovyscript.api.GroovyLog;
+import com.cleanroommc.groovyscript.api.IIngredient;
+import com.cleanroommc.groovyscript.api.documentation.annotations.*;
+import com.cleanroommc.groovyscript.compat.mods.ModSupport;
+import com.cleanroommc.groovyscript.core.mixin.artisanworktables.RecipeRegistryAccessor;
+import com.cleanroommc.groovyscript.helper.recipe.AbstractRecipeBuilder;
+import com.cleanroommc.groovyscript.helper.recipe.RecipeName;
+import com.cleanroommc.groovyscript.registry.AbstractCraftingRecipeBuilder;
+import com.cleanroommc.groovyscript.registry.VirtualizedRegistry;
+import com.codetaylor.mc.artisanworktables.api.ArtisanAPI;
+import com.codetaylor.mc.artisanworktables.api.internal.recipe.*;
+import com.codetaylor.mc.artisanworktables.api.internal.reference.EnumTier;
+import com.codetaylor.mc.artisanworktables.api.recipe.ArtisanRecipe;
+import com.codetaylor.mc.artisanworktables.api.recipe.IArtisanRecipe;
+import com.codetaylor.mc.artisanworktables.api.recipe.requirement.IRequirement;
+import com.codetaylor.mc.artisanworktables.api.recipe.requirement.IRequirementBuilder;
+import com.codetaylor.mc.artisanworktables.modules.worktables.recipe.RecipeTierCalculator;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.ResourceLocation;
+import net.minecraftforge.fml.common.Loader;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+@RegistryDescription(admonition = {
+        @Admonition(type = Admonition.Type.INFO, value = "groovyscript.wiki.artisanworktables.tabletypes"),
+        @Admonition(type = Admonition.Type.DANGER, value = "groovyscript.wiki.artisanworktables.ctcrash")
+})
+public class Tables extends VirtualizedRegistry<IArtisanRecipe> {
+
+    @Override
+    public void onReload() {
+        removeScripted().forEach(r -> {

```

Comment:
I mean, I think adding any `removeRecipe` is too complicated because the recipes have multiple inputs of three different types, multiple outputs of two different types and so on. So the backup will always be empty